apps/dapp-console-api/src/constants/session.ts

```diff
@@ -0,0 +1,23 @@
+import { type IronSession, type SessionOptions } from 'iron-session'
+
+import { envVars } from './envVars'
+
+export const sessionOptions: SessionOptions = {
+  password: envVars.IRON_SESSION_SECRET,
+  cookieName: 'dapp-console-api.auth.session-token',
```

Comment:
it might be a good idea to include our development env as part of the namespace for this so once we have both dev & prod working they don't collide

Reply:
good call!